Contributions/CheckList_for_Safety_Claims_on_a_Generic_Linux_System.md

```diff
@@ -0,0 +1,269 @@
+And because of the variance in the output of the very same toolchain, based on the options selected and the associated values, it is thus necessary for the toolchain to be used with the same parameters that were used during its qualification.
+
+It is therefore necessary, as preliminary step, to verify that the toolchain is qualified accordingly to safety requirements, and used accordingly to its qualification. Lacking this, then it becomes necessary to identify an adequate mitigation strategy.
+
+## **Linux Safety Checklist**
+
+The items presented below are not necessarily coupled with each other, however they are loosely ordered starting from the most fundamental ones, with the intention of creating a "base of safety" for each new one. This doesn't prevent a design where mitigation take a different route.
+However, even if addressed through a fundamentally different approach, they should still be somehow addressed explicitly, even if only to say that they might not be applicable, or that they are handled otherwise.
+
+The analysis covers safety woes, which need to be addressed when making any safety claims.
+
+Safety woes belong primary to two classes:
+* integrity - the safety-related functionality is in a corrupted state\
+In its basic form, integrity mandates the detection of corruption, within an allotted time, so that some pre-defined reaction can happen timely. Stricter integrity requirements would instead dictate that the corruption doesn't impair the functionality. 
+
+* availability - the safety-related functionality becomes unavailable\
+Having any actual availability requirement will percolate in some form on the integrity requirements. High availability doesn't automatically imply that corruption cannot happen, but that it cannot cause interruption of a service. Restarting the affect component or having redundancy are examples of non-preventive mitigation, however any mitigation needs to be evaluated against the specific use-case at hand.
+
+Some of the cases considered might appear as double-failure, which is usually ignored in the context of functional safety, however they are instead cascading failures stemming from a single one, and therefore fully relevant to this analysis.
+
+
+**Items List:**
+
+* [Homogeneity of the test/validation/verification environment](#Homogeneity-of-the-test/validation/verification-environment)
+* [Intra-kernel spatial interference](#Intra-kernel-spatial-interference)
+* [Inter-process interference for resources and side effects](#Inter-process-interference-for-resources-and-side-effects)
+
+### **Homogeneity of the test/validation/verification environment**
+
+This list item is not teribly specific to Linux architecture, however it cannot be overlooked, because doing so can compromise the usefulness of any further work.
```

Comment:
```suggestion
This list item is not specific to Linux architecture, but it cannot be overlooked, because doing so can compromise the usefulness of any further work.
```